pages/en/delegating.mdx

```diff
@@ -83,6 +83,14 @@ Imagine an Indexer has 100,000,000 GRT delegated to them, and their capacity is
 
 Therefore a Delegator should always consider the Delegation Capacity of an Indexer, and factor it into their decision making.
 
+## Delegator FAQs and Bugs 
+
+### MetaMask "Pending Transaction" Bug
+
+*When I try to delegate my transaction in MetaMask appears as "Pending" or "Queued" for longer than expected. What should I do?*
+
+At times, attempts to delegate to indexers via MetaMask can fail and result in prolonged periods of "Pending" or "Queued" transaction attempts. For example, a user may attempt to delegate and run out of ETH to cover the gas costs to complete the transaction, resulting in the transaction attempt displaying as "Pending" in their MetaMask wallet for 15+ minutes. When this occurs, subsequent transactions can be attempted by a user, which pushes the initial transaction into a queue and raises the most recent attempt(s) to "Pending" in place of the initial attempt. In such cases, these transactions can be cancelled in MetaMask, but the transactions attempts will accrue gas fees without any guarantee that subsequent attempts will be successful. The cheapest known fix to this bug is restarting the browsesr (e.g., using "abort:restart" in the address bar), which will cancel all previous attempts without gas being subtracted from the wallet. Several users that have encountered this issue and have reported successful transactions after restarting their browser and attempting to delegate. 
```

Comment:
Some edits for accuracy:

> delegate and run out of ETH to cover the gas costs to complete the transaction

...delegate with an insufficient gas fee, given current prices...

>  which pushes the initial transaction into a queue and raises the most recent attempt(s) to "Pending" in place of the initial attempt

... but these will not be processed until the initial transaction is mined, as transactions for an address must be processed in order.

> the transactions attempts will accrue gas fees without any guarantee that subsequent attempts will be successful

... the cancellation transactions will incur gas fees.

> The cheapest known fix to this bug

A simpler resolution